Product/Production/Adapters/General/CONNECTAdminGUI/src/test/java/gov/hhs/fha/nhinc/admingui/services/impl/SHA2PasswordServiceTest.java

```diff
@@ -1,110 +1,110 @@
-
-import org.junit.Test;
-
-/**
- * The Class SHA2PasswordServiceTest.
- */
-/**
- * @author msw
- *
- */
-public class SHA2PasswordServiceTest {
-
-    /**
-     * Test.
-     *
-     * @throws PasswordServiceException the password service exception
-     * @throws NoSuchAlgorithmException the no such algorithm exception
-     * @throws IOException Signals that an I/O exception has occurred.
-     */
-    @Test
-    public void test() throws PasswordServiceException, NoSuchAlgorithmException, IOException {
-        byte[] candidatePassword = "password".getBytes();
-        byte[] salt = "ABCD".getBytes();
-        byte[] passwordHash = "eFw9+D8egYfAGv1QjUMdVzI9dtvwiH3Amc6XlBoXZj03ebwzuQU8yoYzyLtz40JOn69a7P8zqtT7A6lEyIMBmw=="
-                .getBytes();
-        PasswordService service = getSHA2PasswordService();
-        Assert.assertTrue("Password should match",service.checkPassword(passwordHash, candidatePassword, salt));
-    }
-
-    /**
-     * Test that an incorrect password returns false.
-     *
-     * @throws PasswordServiceException the password service exception
-     * @throws NoSuchAlgorithmException the no such algorithm exception
-     * @throws IOException Signals that an I/O exception has occurred.
-     */
-    @Test
-    public void testNegative() throws PasswordServiceException, NoSuchAlgorithmException, IOException {
-        byte[] candidatePassword = "candidatePassword".getBytes();
-        byte[] salt = "ABCD".getBytes();
-        byte[] passwordHash = "nottherighthash".getBytes();
-        PasswordService service = getSHA2PasswordService();
-        Assert.assertFalse("Password doesn't match",service.checkPassword(passwordHash, candidatePassword, salt));
-    }
-
-    @Test
-    public void generateHash() throws IOException, PasswordServiceException {
-        String salt = "ABCD";// generateRandomSalt();
-        String password = "password";
-        String sha1 = new String(calculateHash(salt.getBytes(), password.getBytes()));
-        Assert.assertTrue("It should less than 100 characters", sha1.length()<=100);
-        Assert.assertEquals("eFw9+D8egYfAGv1QjUMdVzI9dtvwiH3Amc6XlBoXZj03ebwzuQU8yoYzyLtz40JOn69a7P8zqtT7A6lEyIMBmw==", sha1);
-    }
-
-    private byte[] calculateHash(byte[] salt, byte[] password) throws IOException, PasswordServiceException {
-
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        outputStream.write(salt);
-        outputStream.write(password);
-
-        PasswordService service = getSHA2PasswordService();
-        return service.calculateHash(outputStream.toByteArray());
-    }
-
-    /**
-     * Gets the SHA2 password service.
-     *
-     * @return the SHA2 password service
-     */
-    private PasswordService getSHA2PasswordService() {
-        return new SHA2PasswordService();
-    }
-
-}
+/*
+ * Copyright (c) 2009-2017, United States Government, as represented by the Secretary of Health and Human Services.
+ * All rights reserved.
```

Comment:
Please see similar comment above. Undo this file and only change 2009-2017 to reserve file format